net/tstun/wrap_test.go

```diff
@@ -435,22 +432,6 @@ func TestFilter(t *testing.T) {
 			}
 		})
 	}
-
-	inACL := metricInboundDroppedPacketsTotal.Get(dropPacketLabel{Reason: DropReasonACL})
-	inError := metricInboundDroppedPacketsTotal.Get(dropPacketLabel{Reason: DropReasonError})
-	outACL := metricOutboundDroppedPacketsTotal.Get(dropPacketLabel{Reason: DropReasonACL})
-
-	assertMetricPackets(t, "inACL", "3", inACL.String())
-	assertMetricPackets(t, "inError", "0", inError.String())
-	assertMetricPackets(t, "outACL", "1", outACL.String())
```

Comment:
Did you mean to drop these tests rather than migrate them to use the metric registry?

Reply:
Fixed in split out pr

Reply:
Fixed in split out pr